services/cleanup/regular.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from shared.django_apps.reports.models import ReportDetails
+
+from services.cleanup.cleanup import run_cleanup
+from services.cleanup.utils import CleanupResult, CleanupSummary
+
+log = logging.getLogger(__name__)
+
+
+def run_regular_cleanup() -> CleanupSummary:
+    log.info("Starting regular cleanup job")
```

Comment:
I'm assuming these logs are intentional

Reply:
since this is running infrequently, I would like to get a bit more verbose output. Not too much though :-)